src/components/current-query/index.js

```diff
@@ -47,64 +51,74 @@ class ListFacetType extends React.Component {
       }
 
       // Send query based on new state.
-      this.props.onChange(field, values.filter((v, i) => i !== foundIdx));
+      onChange(field, values.filter((v, i) => i !== foundIdx));
     }
   }
 
-  render() {
-    const { searchField } = this.props;
-    return (searchField.value.map((val, i) => (
-      <FacetType
-        key={i}
-        id={i}
-        onClick={() => this.removeListFacetValue(searchField.field, searchField.value, val)}
-      >
-        {/* Add spacing to hierarchical facet values: Type>Term = Type > Term. */}
-        {val.replace('>', ' > ')}
-      </FacetType>
-    )));
-  }
-}
+  return (searchField.value.map((val) => (
+    <FacetType
+      key={searchField.field}
+      id={searchField.field}
```

Comment:
I did this to resolve a "don't use an index in the id" linting error. I don't think it changed/broke any behavior, but it's worth a look.